app/objects/c_adversary.py

```diff
@@ -78,3 +81,6 @@ async def which_plugin(self):
             if await self.walk_file_path(os.path.join('plugins', plugin, 'data', ''), '%s.yml' % self.adversary_id):
                 return plugin
         return None
+
+    def check_repeatable_abilities(self, ram):
```

Comment:
This is a very minor thing, but it might be more efficient and cleaner to just pass a list of abilities extracted from ram rather than the entire ram object. If the general consensus is that it doesn't really matter, that's fine, but I think filtering to abilities when calling the method and renaming ram to ability_list or the like might be cleaner.

Reply:
Works for me, updated in https://github.com/mitre/caldera/pull/2019/commits/c021d41fda2656349e46a08dbd7be1209123ecd5